possibleGraphqlTypes.json

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
The name possible is a bit confusing to me :) Can't it just be named `graphqlTypes.json`?

Reply:
Unsure how to treat this naming.. this is how they want us to use it https://github.com/HedvigInsurance/web-onboarding/pull/382/files#diff-56db900dae2350588c5403a48a41390c684cdf3b11a3ad41fdc7113aaee55fd6R6-R35